Magento_Customer/templates/widget/dob.phtml

```diff
@@ -19,7 +19,7 @@ NOTE: Regarding styles - if we leave it this way, we'll move it to boxes.css. Al
 automatically using block input parameters.
 */
 
-$fieldCssClass = 'field date field-' . $block->getHtmlId();
+$fieldCssClass = 'margin-top-m field date field-' . $block->getHtmlId();
```

Comment:
Do we use this classes in templates? As I wonder if we shouldn't use here some field/form related class and add styles on some molecul/organism level? I mean if we use for fields larger/smaller spacing wouldn't we need to change this class?

Reply:
All inputs have class which adds margin bottom, I've added margin-top to add more space between Sign up to newsletter checkbox and DOB input. I can remove it but then I have to add margin-bottom for checkbox.  Not sure what is better

Reply:
I mean we could create widget-dob class or something specific for this template so it can be styled via css. But I see in few places margin-top-m in other templates so we can leave ii like it is. thanks